session19/src/Challenge/Person.java

```diff
@@ -0,0 +1,28 @@
+package Challenge;
+
+import java.util.Objects;
+
+public class Person {
+
+    String name;
+    int age;
+    String email;
```

Comment:
these should be private (not default/package protected) and create getters and setters for them if you want to assign/retrieve their values (use Encapsulation - OOP concept).